p2psrv/options.go

```diff
@@ -40,11 +40,6 @@ type Options struct {
 
 	KnownNodes Nodes
 
-	// BootstrapNodes are used to establish connectivity
-	// with the rest of the network using the V5 discovery
-	// protocol.
-	BootstrapNodes Nodes
-
```

Comment:
This seems to be redundant given that the BootstrapNodes = CachedNodes + Bootstrap nodes that was set in the utils files